utils/data.py

```diff
@@ -23,33 +24,9 @@ async def on_message(self, msg):
         await self.process_commands(msg)
 
 
-class HelpFormat(DefaultHelpCommand):
-    def get_destination(self, no_pm: bool = False):
-        if no_pm:
-            return self.context.channel
-        else:
-            return self.context.author
-
-    async def send_error_message(self, error):
-        destination = self.get_destination(no_pm=True)
-        await destination.send(error)
-
-    async def send_command_help(self, command):
-        self.add_command_formatting(command)
-        self.paginator.close_page()
-        await self.send_pages(no_pm=True)
-
-    async def send_pages(self, no_pm: bool = False):
-        try:
-            if permissions.can_handle(self.context, "add_reactions"):
-                await self.context.message.add_reaction(chr(0x2709))
-        except discord.Forbidden:
-            pass
-
-        try:
-            destination = self.get_destination(no_pm=no_pm)
-            for page in self.paginator.pages:
-                await destination.send(page)
-        except discord.Forbidden:
-            destination = self.get_destination(no_pm=True)
-            await destination.send("Couldn't send help to you due to blocked DMs...")
+class CustomHelp(commands.MinimalHelpCommand):
+    async def send_pages(self):
+        destination = self.get_destination()
+        for page in self.paginator.pages:
+            embed = discord.Embed(description=page)
+            await destination.send(embed=embed)
```

Comment:
Please turn this back to text instead, the reason it is text is to avoid the embed errors that some server don't allow

Reply:
aww man, will do